provisionerd/runner/runner.go

```diff
@@ -248,9 +251,6 @@ func (r *Runner) doCleanFinish(ctx context.Context) {
 		completedJob *proto.CompletedJob
 	)
 
-	ctx, span := r.startTrace(ctx, tracing.FuncName())
-	defer span.End()
```

Comment:
Not sure why this was removed but seems like it should be fine.

Reply:
I was originally going in to change this but then I found the `-no-color` issue so I bundled them up.

Reply:
Having `doCleanFinish` be a super long span didn't seem that useful since everything else inside of it was already traced.
![image](https://user-images.githubusercontent.com/6332295/199075639-b3a5a03f-0d18-4260-8155-b172176b6456.png)

Reply:
I see what you mean. I guess it gives a better idea of how the code is structured, but not a necessity if you feel cleanliness wins here 👍🏻.